wgs/workflows/germline_calling_consensus/consensus_tests.py

```diff
@@ -1,126 +1,641 @@
+import random
+import vcf
+import gzip
+import os
+import consensus
+import pypeliner
+from shutil import copyfile
+import csv
+import pandas as pd
+from collections import namedtuple
+
+def _get_test_record():
+    '''
+    make a dummy, correctly formatted snv with correct data
+    Parameters
+    ---------
+    Returns
+    -------
+    '''
+    alt = random.choice(["A", "T", "C", "G"])
+    ref = random.choice(["A", "T", "C", "G"])
+    chrom = random.choice(list(map(str, list(range(1, 23)) + ["X"])))
+    pos = random.randint(0, 100000)
+
+    test_record = [1, '.', 'nan', 1, 2, 3]
+    return chrom, pos, ref, alt, test_record
+
+def _get_test_model_call():
+    #make example  call
+    Call = namedtuple("Call", "DP RO AO RC AC AD")
+    sample_data = vcf.model._Call(None, "sample_label", Call(1,1,2,1,2, [1,2]) )
+
+    freebayes_museq_rtg_example = vcf.model._Record("1", 10, 1, "A", [None], 1, 1, 
+        None, None, None, [sample_data])
+
+    samtools_example = vcf.model._Record("1", 10, 1, "A", [None], 1, 1, {"DP":1}, 
+        None, None, [sample_data])
+
+    return freebayes_museq_rtg_example, samtools_example
+
+##################################
+# test consensus.snv_consensus() #
+##################################
+
+def test_snv_consensus_case_1():
+    '''
+    test  consensus.snv_consensus() with identical record between museq/freebayes
+    Parameters
+    ---------
+    Returns
+    -------
+    '''
+    chrom, pos, ref, alt, record = _get_test_record()
+
+    test_record = {(chrom, pos, ref, alt): recordd}
+
+    museq = test_record
+    freebayes = test_record
+    rtg = []
+    samtools = []
+
+    consensus_data = consensus.snv_consensus(museq, freebayes, rtg, samtools)
+    
+    consensus_data = pd.DataFrame(consensus_data, 
+        columns = ["chrom", "pos", "ref", "alt", "id_counter", "qual", "filter", "nr", "na", "nd"]
+    )
+
+    consensus_data = consensus_data.astype({"chrom": "str"})
+    
+    assert not consensus_data[(consensus_data.pos==pos) & (consensus_data["chrom"]==chrom)].empty
```

Comment:
ideally I'd check that there's one record in consensus.. 

and confirm that it matches the input record